helm/Chart.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: harness-helm
+description: A Helm chart for Kubernetes
+maintainers:
+  - name: Hrittik Roy
+    email: hrittik.roy@harness.io
```

Add a generic email `community-engineering@harness.io`
```suggestion
    email: community-engineering@harness.io
```